src/Controller/MainPageController.py

```diff
@@ -846,17 +844,14 @@ def edit_mode(self):
                 if df.at[i, 'PID'] == self.dataset[0].PatientID:
                     check = True
             if not check:
-                # the sensitive data for this patient is missing so no
-                # editing can be performed
-                button_reply = QtWidgets.QMessageBox.warning(
-                    self,
-                    "Error Message",
-                    "The software has no previous records of this patient.\n"
-                    "If you wish, you can create a new clinical data file by "
-                    "\ndeleting the current one from the directory and "
-                    "reloading \nthe patient files.",
-                    button0=QtWidgets.QMessageBox.Ok)
-                if button_reply == QtWidgets.QMessageBox.Ok:
+                # the sensitive data for this patient is missing so no editing can be performed
```

Comment:
Please keep the edit from current files, and only change `button0=QtWidgets.QMessageBox.Ok` to `QtWidgets.QMessageBox.Ok`. The current texts comply with PEP 8 standard, only missing the typo `button0` fix.